tools/configure/util.ml

```diff
@@ -144,9 +144,12 @@ let numeric_prefix_list s =
   | [v1;v2;""] -> [v1;v2;"0"] (* e.g. because it ends with ".beta" *)
   | v -> v
 
+let debug_version_detection = false
```

Comment:
We should either actually remove the code or make this controllable from the command line imo

Reply:
I'm for completely removing it, if it's really needed to be kept around we can always fish it out of the git history.

Reply:
It is actually usual for these kind of system APIs to actually have debug parameters, and keep the unexposed to the final user.

Reply:
I'm also in favor of completely removing it.